contracts/core/EntryPointSimulations.sol

```diff
@@ -170,13 +170,12 @@ contract EntryPointSimulations is EntryPoint, IEntryPointSimulations {
     //make sure depositTo cost is more than normal EntryPoint's cost.
     // empiric test showed that without this wrapper, simulation depositTo costs less..
     function depositTo(address account) public override(IStakeManager, StakeManager) payable {
-        uint x;
-        assembly {
-            //some silly code to waste ~200 gas
-            x := exp(mload(0),100)
-        }
-        if (x == 123) {
-            return;
+        unchecked{
+            uint g = gasleft();
```

Comment:
What's going on here?

Reply:
Its a code to waste some gas... we must make sure that the simulation contract always takes more gas, otherwise, it can be used to attack the bundler.
The previous "gas waster" didn't pass the tests, but wasn't easily adaptable, so I changed it to waste gas in a loop.
Strangely, the different functions (with/without value, using fallback or "depositTo") have different delta from the real entrypoint code. so I had to make sure it is above for all.